pytest-selenium/utils/utility.py

```diff
@@ -0,0 +1,14 @@
+import random
+from random import choice
```

Comment:
You should only need the top import. `from random import choice` is redundant.

Reply:
Updated